docs/user-guide/index.md

```diff
@@ -8,7 +8,7 @@ The Kubernetes **Guides** can help you work with various aspects of the Kubernet
 
 * The Kubernetes [User Guide](#user-guide-internal) can help you run programs and services on an existing Kubernetes cluster.
 * The [Cluster Admin Guide](/docs/admin/) can help you set up and administrate your own Kubernetes cluster.
-* The [Developer Guide](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel) can help you either write code to directly access the Kubernetes API, or to contribute directly to the Kubernetes project.
+* The [Developer Guide] can help you either write code to directly access the Kubernetes API, or to contribute directly to the Kubernetes project.
```

Comment:
literal href values, like literal bibliographic references, clutter copy text.  Markup systems always offer a way to pull them out (e.g. \bibitem in latex) to make the copy easier to read/edit.

Also, bibliographies make it easier to notice url mistakes, notice inconsistencies in anchor text, change links without having to edit multiple spots, notice glaring reference omissions (e.g. pointing to distant leaves without mentioning main docs), etc.

I want to (lazily - as we fix bad links, edit copy for other reasons, etc.) move to bibliography style references in all docs so we get these benefits across the board.